hack/externalnode/install-vm.sh

```diff
@@ -64,7 +64,7 @@ ANTREA_DOCKER_IMAGE="${ANTREA_DOCKER_REGISTRY}:${ANTREA_VERSION}"
 NODE_NAME="$(hostname)"
 OS_NAME=""
 
-declare -a SUPPORTED_OS=("ubuntu 18.04" "ubuntu 20.04" "rhel 8.4")
+declare -a SUPPORTED_OS=("ubuntu 18.04" "ubuntu 20.04" "ubuntu 22.04" "rhel 8.4")
```

Comment:
I have tried 22.04 with no incompatibility found. external-node.md doc updated

Reply:
This change won't be known without looking at the code change. It's not reflected in neither title nor description. Please move it to a separate PR which declares the support of ubuntu 22.04 for ExternalNode so we can include it in release note properly, instead of refering to a PR about e2e test.

Reply:
not addressed

Reply:
updated